lua/nui/popup/border.lua

```diff
@@ -428,6 +430,7 @@ function Border:init(popup, options)
     border = "none",
     focusable = false,
     zindex = self.popup.win_config.zindex - 1,
+    anchor = self.popup._.anchor,
```

Comment:
Can we directly read form `self.popup.win_config.anchor` instead? And get rid of `self.popup._.anchor` altogether? 🤔

Reply:
I don't think so, since Border overrides it? So the next mount it would be wrong

Reply:
`:init` is only called once.

So if the popup has `"simple"` border, then `popup.win_config.anchor` will never get overridden: https://github.com/MunifTanjim/nui.nvim/blob/f2204952ba670372de507bdd446c6a14f821ac73/lua/nui/popup/border.lua#L305-L310

And if the popup has `"complex"` border, `popup.win_config.anchor` will always have `"NW"`, and `border.win_config.anchor` will have the `anchor` the popup was originally created with.

For subsequent `:mount`/`:show` calls, it should keep working as expected.

Reply:
Yep, you're right. Updated!